lib/src/sql/function.rs

```diff
@@ -168,6 +168,9 @@ impl Function {
 				fnc::run(ctx, opt, txn, doc, s, a).await
 			}
 			Self::Custom(s, x) => {
+				// Check that namespace and database are set to prevent a panic
+				opt.valid_for_ns()?;
```

Comment:
Just realised, we shouldn't need this line, as the `opt.valid_for_db()?;` line calls this internally.
```suggestion
```

Reply:
Totally missed that, thank you! Fixed in [5c7c029](https://github.com/surrealdb/surrealdb/pull/3297/commits/5c7c029017d0aecd5be8fc715d07ab492bbea1f7).